pkg/registry/core/service/storage/rest_test.go

```diff
@@ -67,6 +69,34 @@ type serviceStorage struct {
 	Err                error
 }
 
+type testNetworkGetter struct {
+	ipamType string
+	err      error
+}
+
+func (n testNetworkGetter) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
```

Comment:
param "name" and "ctx" are never used. Is it intentional?

BTW, this struct is especially created for mizar ( or say external IPAM type) test. It is better to rename "testNetworkGetter" to indicate it.

Reply:
ctx & name not used at all as it is a test dummy fixture.
change the struct type name to indicates that it is used as a dummy.